--- ./check_ftp_fully	2010-03-16 06:12:56.000000000 +0200
+++ ./check_ftp_fully	2015-03-25 11:57:12.000000000 +0200
@@ -102,7 +102,7 @@
 fi
 
 # Get test
-${LFTP} -u ${USERNAME},${PASSWORD} -p${PORT} -e "LCD ${FILEDIR}; GET ${FILENAME}; quit" ${HOSTNAME} &> /dev/null
+${LFTP} -u ${USERNAME},${PASSWORD} -p${PORT} -e "GET ${FILENAME}; quit" ${HOSTNAME} &> /dev/null
 result=$?
 if [ ${result} != 0 ]; then
     f_cri "FTP check failed when trying to get a file from a directory."
